### PR TITLE
Implement LIC0

### PR DESCRIPTION
**Pull request by andreaskth**
This PR implements LIC 0 and its corresponding tests.

Fixes: #6

### CODE REVIEW

**Comment by andreaskth** [comment]
Anyone know why the base-branch was set to `main` instead of `development`? I'm positive I was on the `development`-branch when I ran `git checkout -b issue-6-lic-0`...

**Comment by elmaxe** [comment]
> Anyone know why the base-branch was set to `main` instead of `development`? I'm positive I was on the `development`-branch when I ran `git checkout -b issue-6-lic-0`...

It automatically assumes you want to merge to master instead of development, has been like that for all my PRs

File: src/test/java/LIC0Test.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import org.junit.jupiter.api.Test;
```
Comment (elmaxe) [inline src/test/java/LIC0Test.java:1]: Wrong indentation in this file. It should use two-space indenation, this one uses more

**Comment by K2017** [comment]
main is just the default branch, don't you have the option to change the base branch before submitting the request?

**Review by elmaxe — CHANGES_REQUESTED** [review]
See comments

**Comment by andreaskth** [comment]
> main is just the default branch, don't you have the option to change the base branch before submitting the request?

Yeah I almost definitely can, I just assumed it would be correct since the branch was created correctly, so didn't look too closely. Was easy to switch though so no problems

File: src/test/java/LIC0Test.java (hunk 1 above)
Comment (andreaskth) [reply src/test/java/LIC0Test.java:1]: Fixed!

**Review by elmaxe — APPROVED** [review]

**Review by K2017 — APPROVED** [review]

**Pull request merged**